bedrock/firefox/templates/firefox/new/desktop/base.html

```diff
@@ -6,19 +6,15 @@
 
 {% extends "firefox/base/base-protocol.html" %}
 
-{% block page_title_prefix %}{{ ftl('firefox-desktop-download-meta-title-prefix') }} — {% endblock %}
-
-{%- block page_title -%}{{ ftl('firefox-desktop-download-meta-title-v2', fallback='firefox-desktop-download-meta-title') }}{%- endblock -%}
-
-{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+{# Bug 1438302,  Issue 13019: Avoid duplicate content for English pages. #}
 {%- block page_title_full -%}
-  {% if LANG == 'en-CA' %}
```

Comment:
Note for reviewer: older `en-CA` title is being removed (Adria think we shouldn't really need them)